teps/0053-tekton-catalog-pipeline-organization.md

```diff
@@ -0,0 +1,477 @@
+---
+status: proposed
+title: Tekton Catalog Pipeline Organization
+creation-date: '2021-02-22'
+last-updated: '2021-02-22'
+authors:
+  - '@piyush-garg'
+  - '@PuneetPunamiya'
+---
+
+# TEP-0053: Tekton Catalog Pipeline Organization
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+  - [Use Cases (optional)](#use-cases-optional)
+- [Requirements](#requirements)
+- [Proposal](#proposal)
+  - [Support-Tiers](#support-tiers)
+  - [Versioning Resources](#versioning-resources)
+  - [Ownership](#ownership)
+  - [Organzation](#organization)
+  - [Requirements &amp; Guidelines](#requirements--guidelines)
+  - [Deprecation & Removal strategy](#deprecation--removal-strategy)
+- [Design Details](#design-details)
+- [Test Plan](#test-plan)
+- [Design Evaluation](#design-evaluation)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+- [Infrastructure Needed (optional)](#infrastructure-needed-optional)
+- [Upgrade &amp; Migration Strategy (optional)](#upgrade--migration-strategy-optional)
+- [References (optional)](#references-optional)
+
+## Summary
+
+Tekton’s mission is to be the industry standard cloud-native CI/CD platform and ecosystem. At this point in the project Tekton is providing a lot of value for folks building CI/CD systems by giving them out of the box scalable, cloud native, serverless, execution. However Tekton also wants to reduce the fragmentation in the CI/CD space by creating reusable resources that folks building Pipelines can use and share. This is where the Tekton catalog comes in!
+
+The Tekton catalog is a collection of blessed Tekton resources that can be used with any system that supports the Tekton API
+
+## Motivation
+
+As the Tekton Catalog grows, it is important to define an organisation for pipeline as did for task in [TEP-0003: Tekton Catalog Organization](https://github.com/tektoncd/community/blob/main/teps/0003-tekton-catalog-organization.md), a set of rules and clear ownership that make catalog successful and sane in the long-term. This should also help other projects to define their own catalog that would satisfy the catalog contract.
+
+### Goals
+
+<!--
+List the specific goals of the TEP.  What is it trying to achieve?  How will we
+know that this has succeeded?
+-->
+
```

Comment:
we should include some goals here too, maybe something like:

* Define the process and standards for submitting Pipelines to the catalog, as we have for Tasks
* Create a standard around how to refer to Tasks in the catalog from Pipelines in the catalog